recipes/cmor/meta.yaml.in

```diff
@@ -0,0 +1,40 @@
+package:
+    name: cmor
+    version: @VERSION@
+
+source:
+    git_rev: @UVCDAT_BRANCH@
+    git_url: git://github.com/PCMDI/cmor.git
+
+build:
+  number: @BUILD_NUMBER@
+
+test:
+  requires:
+    - python
+    - gcc [osx]
+  imports:
+    - cmor
+
+requirements:
+  build:
+    - python
+    - ossuuid 1.6.2
+    - udunits2
+    - hdf5 1.8.17
```

Comment:
@dnadeau4 hdf5 1.8.18 please

Reply:
@dnadeau4 you can get recipe from http://github.com/uv-cdat/conda-recipes

Reply:
Done in master